### PR TITLE
Fix monodepth2 issue

### PR DESCRIPTION
**Pull request by KuangHaofei**
Fix two issues of monodepth2:
1. posenet hybridiz issue: https://github.com/dmlc/gluon-cv/issues/1460
2. lr decay issue: https://github.com/dmlc/gluon-cv/issues/1462

### CODE REVIEW

**Comment by mli** [comment]
Job PR-1472-2 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1472/2/index.html 
Code coverage of this PR:  vs. Master:

File: scripts/depth/options.py [hunk 1]
```diff
@@ -193,12 +193,6 @@ def parse(self):
         self.options = self.parser.parse_args()
 
         # logging and checkpoint saving
```
Comment (bryanyzhu) [inline scripts/depth/options.py:195]: If the code below is deleted, maybe this line needs to be deleted as well.

**Review by bryanyzhu — APPROVED** [review]
Looks good to me. Thank you for resolving the issues.

**Review by zhreshold — APPROVED** [review]

**Comment by mli** [comment]
Job PR-1472-3 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1472/3/index.html 
Code coverage of this PR:  vs. Master:

**Comment by pandyadhatri** [comment]
XNetError                                Traceback (most recent call last)
<ipython-input-70-72280502c8a8> in <module>()
     20         # AutoGrad
     21         with ag.record():
---> 22             output = [net(X) for X in data]
     23         #    loss = [loss_fn(yhat, y) for yhat, y in zip(output, label)]
     24             loss_fn = gluon.loss.SoftmaxCrossEntropyLoss(sparse_label=False)

8 frames
/usr/local/lib/python3.7/dist-packages/mxnet/base.py in check_call(ret)
    244     """
    245     if ret != 0:
--> 246         raise get_last_ffi_error()
    247 
    248 

MXNetError: MXNetError: Shape inconsistent, Provided = [64,784], inferred shape=(64,4816896)

**Pull request merged**